xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -175,7 +175,7 @@ void CPeripheralCecAdapter::Announce(AnnouncementFlag flag, const char *sender,
   else if (flag == GUI && !strcmp(sender, "xbmc") && !strcmp(message, "OnScreensaverActivated") && m_bIsReady)
   {
     // Don't put devices to standby if application is currently playing
-    if ((!g_application.m_pPlayer->IsPlaying() && !g_application.m_pPlayer->IsPaused()) && m_configuration.bPowerOffScreensaver == 1)
+    if (!g_application.m_pPlayer->IsPausedPlayback() && m_configuration.bPowerOffScreensaver == 1)
```

Comment:
so basically all that changed is that `IsPlaying()` stayed the same, but `IsPaused()` got changed to `IsPlaybackPaused()`? What does this new `IsPlaybackPaused()` do different from `IsPaused()` which is apparently still present or this wouldn't have compiled?

Don't we have an alternative in our beloved CApp that tells whether the player is active? Because that's all what it's supposed to do, don't put the TV to standby while the player is active.